xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+     You can use different versions of required packages in the container than
+     the host system uses without having problems with dependencies.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     You can run several instances of the same application that are completely
+     independent from each other.
+    </para>
+   </listitem>
+  </itemizedlist>
+  </sect2>
+  <sect2 xml:id="sec.docker.concept.images-applications.build">
+   <title>Advantages of &deng; to Build Applications</title>
+   <para>
+    Using &deng; to build applications has the following advantages:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      You can prepare a complete building image.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Your build always runs in the same environment.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Developers can test their code in the same environment as used in
+      production.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      You can set up an automated building process.
+     </para>
+    </listitem>
+   </itemizedlist>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="sec.docker.concept.architecture_deng">
+  <title>&deng; Architecture</title>
+  <para>
+   &deng; uses a client/server architecture. You can use the <emphasis>CLI
+    client</emphasis> to communicate with the <emphasis>daemon</emphasis>. The
+   <emphasis>daemon</emphasis> then performs operations with containers and
+   manages images locally or in registry. The <emphasis>CLI client</emphasis>
+   can run on the same server as the host daemon or on a different machine. The
+   <emphasis>CLI client</emphasis> communicates with the
+   <emphasis>daemon</emphasis> by using network sockets. The architecture is
+   depicted in <xref linkend="Docker_architecture"/>.
+  </para>
+  <remark>toms 2019-02-04: The font size in the following picture is too
+   small. Please increase the font size with Inkscape so it's more readable.
+  </remark>
+  <figure xml:id="Docker_architecture">
+   <title>The &deng; Architecture</title>
+   <mediaobject>
+    <imageobject>
+     <imagedata fileref="docker_architecture.png" format="PNG" width="95%"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+ </sect1>
+
+ <sect1 xml:id="sec.docker.concept.definition_dreg">
+  <title>&dreg;</title>
+
+  <para>
+   The &dreg; is an open-source project created by Docker Inc. It
+   allows the storage and retrieval of &docker; images. By running a local
+   instance of the &dreg; it is possible to completely avoid the usage of
+   &dhub;.
+  </para>
+
+  <para>
+   &dreg; is also used by &dhub;. However, &dhub;,
+   as seen from the user perspective, is made of the following parts at least:
+  </para>
+
+  <remark>toms 2019-02-05: Not sure if it is better to have separate sections
+  about terminology instead of one single one at the beginning.
+  Maybe you need to rephrase it to make it clear that the terms below refers
```

Comment:
ATM (while reading this PR online), I'm looking for language issues mostly. I will focus on the structure in a second review and then also look at this question in detail. I need to do the structure review in a printed version of the guide though, after those issues have been fixed.